scripts/congrats-bot.mjs

```diff
@@ -0,0 +1,61 @@
+import { setOutput } from '@actions/core';
+
+const { COMMIT_AUTHOR, COMMIT_ID, COMMIT_MESSAGE } = process.env;
+setDiscordMessage(COMMIT_AUTHOR, COMMIT_ID, COMMIT_MESSAGE);
+
+/**
+ * @param {string} author The name of the commit author
+ * @param {string} id The commit ID
+ * @param {string} commitMsg A full commit message
+ */
+function setDiscordMessage(author, id, commitMsg) {
+	const commitMessage = commitMsg
+		.split('\n')
+		.shift()
+		.replaceAll('`', '');
+	
+	const coAuthors = commitMsg
+		.split('\n')
+		.slice(2)
+		.filter((line) => line.match(/Co-authored-by: (.+) <.+>/))
+		.map((line) => line.match(/Co-authored-by: (.+) <.+>/)[1]);
+
+	let coAuthorThanks = '';
+	if (coAuthors.length) {
+		const names = makeList(coAuthors);
+		coAuthorThanks = '\n' + getCoAuthorsMessage(names);
+	}
+
+	setOutput(
+		'DISCORD_MESSAGE',
+		`**Merged!** ${author}: [\`${commitMessage}\`](<https://github.com/withastro/docs/commit/${id}>)${coAuthorThanks}`
```

Comment:
```suggestion
		` 🎉**Merged!**🎉  ${author}: [\`${commitMessage}\`](<https://github.com/withastro/docs/commit/${id}>)${coAuthorThanks}`
```

I think maybe some small emojis around the **Merged** statement would add a nice touch